app/src/main/java/org/oppia/android/app/home/promotedlist/PromotedStoryListView.kt

```diff
@@ -12,6 +14,8 @@ import org.oppia.android.app.shim.ViewBindingShim
 import org.oppia.android.app.shim.ViewComponentFactory
 import javax.inject.Inject
 
+const val PROMOTED_STORY_LIST_VIEW_TAG = "PromotedStoryListView"
```

Comment:
Should be private if it's not needed outside this file.

Reply:
Done